autoload/fzf/vim.vim

```diff
@@ -1230,8 +1230,10 @@ function! s:commits(range, buffer_local, args)
 
   if !s:is_win && &columns > s:wide
     let suffix = executable('delta') ? '| delta --width $FZF_PREVIEW_COLUMNS' : ''
+    let s:orderfile = tempname()
+    call writefile([current], s:orderfile)
```

Comment:
Is there any reason you use `s:` variable instead of a plain local variable?

Reply:
No, I copied that from somewhere else in your script since I'm not very familiar with vimscript. I will remove it.